prefect_kubernetes/jobs.py

```diff
@@ -153,7 +153,7 @@ def kubernetes_orchestrator():
 async def patch_namespaced_job(
     kubernetes_credentials: KubernetesCredentials,
     job_name: str,
-    body: V1Job,
+    job_updates: V1Job,
```

Comment:
Since all the other ones are like `new_job` with the adjective as a prefix, maybe this can be `updated_job`

```suggestion
    job_updates: V1Job,
```

Reply:
This task only partially updates the job, so I think that t`job_updates` is more descriptive of the provided functionality.